README.md

```diff
@@ -2,15 +2,15 @@
 PaperLib is a plugin library for interfacing with Paper specific APIs (such as async chunk loading), with graceful fallbacks maintaining compatibility with both the Bukkit and Spigot API's.
 
 ## API
-All API can be found as static util methods on the `PaperLib` class.
+All API methods can be found as static in the `PaperLib` class.
```

Comment:
I preferred the previous expression, methods cannot "be found" as static.

Reply:
Should be fixed now!